independent-publisher-connectors/Airtable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,490 @@
+                      },
+                      "createdTime": {
+                        "type": "string",
+                        "description": "Record Created Time"
+                      }
+                    }
+                  },
+                  "description": "Records"
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "Base ID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Base ID",
+            "x-ms-summary": "Base ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Table",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Table",
+            "x-ms-summary": "Table",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "filterByFormula",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "filterByFormula",
+            "x-ms-summary": "filterByFormula"
+          },
+          {
+            "name": "maxRecords",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-visibility": "advanced",
+            "description": "maxRecords",
+            "x-ms-summary": "maxRecords"
+          },
+          {
+            "name": "pageSize",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-visibility": "advanced",
+            "description": "pageSize",
+            "x-ms-summary": "pageSize"
+          },
+          {
+            "name": "view",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "view",
+            "x-ms-summary": "view"
+          },
+          {
+            "name": "cellFormat",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "cellFormat",
+            "x-ms-summary": "cellFormat"
+          },
+          {
+            "name": "timeZone",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "timeZone",
+            "x-ms-summary": "timeZone"
+          },
+          {
+            "name": "userLocale",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "userLocale",
+            "x-ms-summary": "userLocale"
+          }
+        ],
+        "summary": "List Records",
+        "description": "List Records",
```

Comment:
@last72 The description value can be improved as it is same as summary value.

Documentation - [Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values